docs/source/reference/metrics.rst

```diff
@@ -181,3 +181,58 @@ Configure your InfluxDB and Telegraf InfluxDB output as usual, then on the Stats
 .. literalinclude:: /_includes/_telegraf_statsd_input.toml
     :language: toml
     
+Pushing metrics to Prometheus via the statsd_exporter
+=====================================================
+
+It is also possible to make the metrics available for Prometheus and other monitoring solutions that are able to scrape Prometheus targets (i.e. Zabbix). Prometheus provides a docker service called `statsd exporter <https://github.com/prometheus/statsd_exporter>` which receives data in the StatsD format and acts as a scrape target for Prometheus.
+
+st2 configuration:
+
+.. code-block:: ini
+
+    [metrics]
+    driver = statsd
+    # Optional prefix which is prepended to each metric key. E.g. if prefix is
+    # "production" and key is "action.executions" actual key would be
+    # "st2.production.action.executions". This comes handy when you want to
+    # utilize the same backend instance for multiple environments or similar.
+    
+    # statsd collection and aggregation server address
+    host = <statsd_exporter_address>
+    # statsd collection and aggregation server port
+    port = 8125
+
+Docker configuration
+--------------------
+
+The minimal requirement is a started ``statsd exporter`` container with port 8125/udp and 9102/tcp exposed unless you run StackStorm in containers in the same network.
+
+Port 8125/udp is the default port where the container receives the StatsD metrics from StackStorm. Port 9102/tcp is the default port where the statsd exporter exposes the web interface and the Prometheus metrics.
+
+Example docker-compose.yml snippet:
```

Comment:
I think it's best to omit from the docs implementation/deployment details for Prometheus metrics like Docker.
Otherwise it looks like a tutorial.
Instead highlight the https://github.com/prometheus/statsd_exporter repository, the project itself, what problems it solves and configs used.

It makes sense to mention that there is a docker helper and referencing a link to that, but avoid describing docker example in deep details.

Reply:
Ok, I agree. Too much details increase may cause issues and support requests which are actually not related to or part of the core product. I'll rework that part.

Reply:
@armab I addressed your recommendations and pushed a new version. Let me know if that works for you, too.

Reply:
Please check the other syntax issues highlighted below in the conversation. Thanks!